runtime/doc/lsp.txt

```diff
@@ -244,15 +249,18 @@ For |lsp-notification|, each |lsp-handler| has this signature: >
             {err}       (nil)
                             This is always `nil`.
                             See |lsp-notification|
-            {method}    (string)
-                            The |lsp-method| name.
-            {params}    (Params)
+            {result}    (Result)
                             This contains the `params` key of the notification.
                             See |lsp-notification|
-            {client_id} (number)
-                            The ID of the |vim.lsp.client|
-            {bufnr}     (nil)
-                            `nil`, as the server doesn't have an associated buffer.
+            {context}   (table)
```

Comment:
ctx? also the function signature on line 246 wasn't updated.

Reply:
Fixed